### PR TITLE
Pass base image through pipeline

### PR DESCRIPTION
**Pull request by cderb**
Modify tuning pipeline to pass base images
add mi300 to parse args
Update Logic for CI

### CODE REVIEW

File: .jenkins/perf_compile [hunk 1]
```diff
@@ -39,7 +39,7 @@ pipeline {
     string(name: 'db_user', defaultValue: "${PIPELINE_USER}", description: 'Username for the databse')
     string(name: 'db_password', defaultValue: "${PIPELINE_PWD}", description: 'Password for the user')
     string(name: 'docker_registry', defaultValue: "${headnode}:5000", description: 'Name of the docker registry for pushing images')
-    string(name: 'base_image', defaultValue: '', description: 'Put a fully qualified docker name here to use (optional)')
+    //string(name: 'base_image', defaultValue: '', description: 'Put a fully qualified docker name here to use (optional)')
```
Comment (alexandraBara) [inline .jenkins/perf_compile]: comment? I see lots of comments left here, is there a reason for leaving them im?

Comment (cderb) [reply .jenkins/perf_compile]: removed them

File: Dockerfile [hunk 2]
```diff
@@ -90,25 +90,25 @@ ENV UBSAN_OPTIONS=print_stacktrace=1
...
 ARG MIOPEN_DEPS=/opt/rocm
 
 # Install dependencies # included in rocm/miopen:ci_xxxxxx
-#RUN cmake -P install_deps.cmake --prefix $MIOPEN_DEPS
-#RUN CXXFLAGS='-isystem $PREFIX/include' cget install -f ./mlir-requirements.txt
+RUN . /env; if [ -z $NO_ROCM_INST ]; then\
+        pip install cget; \
+        pip install https://github.com/pfultz2/rclone/archive/master.tar.gz; \
+        cmake -P install_deps.cmake --prefix cget; \
+        cp -r cget/* /opt/rocm/.; \
```
Comment (Dmantri98) [inline Dockerfile]: When I built this version of the dockerfile successfully I switched lines 109 and 110. This was due to dev-requirements not being moved to /opt/rocm/, the correct prefix path.

**Review by Dmantri98 — APPROVED** [review]

**Review by Dmantri98 — APPROVED** [review]

**Review by iq136boy — APPROVED** [review]
Looks good to me.

**Review by alexandraBara — APPROVED** [review]

**Pull request merged**